apollo-relay/src/relay.rs

```diff
@@ -0,0 +1,24 @@
+//! Main entry point for CLI command to start relay.
+use apollo_relay::relay::ReportRelay;
+use clap::Parser;
+
+const DEFAULT_LISTEN: &str = "0.0.0.0:50051";
```

Comment:
Would it make sense for the default to be a kernel-assigned port rather than a fixed one?

Reply:
Maybe, but we'd need a way to convey the actual port assigned to the client. That would undermine the default configuration approach I'm using at the moment. I'll think about it some more...